observer/tests/test_cli.py

```
@@ -0,0 +1,213 @@
+    def test_missing_log_file(self, runner, obs_dir):
+        result = runner.invoke(main, ["logs"])
+
+        assert result.exit_code != 0
+        assert "not found" in result.output.lower()
+
+    def test_exec_args(self, runner, obs_dir):
+        log_file = obs_dir / "observer.log"
+        log_file.write_text("test log line\n")
+
+        with patch("observer.cli.os.execvp") as mock_exec:
+            result = runner.invoke(main, ["logs", "-n", "50"])
```

Ruff (0.15.1)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)